Provision of edits for Enums DM (#3437)
Codecademy/docs: content/discrete-math/concepts/enumeration/enumeration.md

**Counting Theory** is a part of discrete mathematics that deals with counting the number of possible outcomes of a given event. Being an essential element of probability, it tries to answer an abstract question, "How many?"

## Preliminaries: On Choices

It is said that in order to combine numbers correctly, it's best to look for the words `AND` and `OR` in the description of the sequence of choices available.

Some notes on `"AND"`:

> - corresponds to the Cartesian product of sets
> - If you choose one element of the set A AND you choose one element of the set B, then this is equivalent to choosing one element of the `Cartesian product of A and B`
> 
> Some notes on `"OR"`:
> 
> - corresponds to the union of sets
> - If you choose one element of the set A OR you choose one element of the set B, then this is equivalent to choosing one element of the `union of A and B`

Reviewer: ```suggestion
- Corresponds to the Cartesian product of sets.
- If one element of the set A `AND` one element of the set B is chosen, then this is equivalent to choosing one element of the `Cartesian product of A and B`.

Some notes on `OR`:

- Corresponds to the union of sets.
- If an element of the set A `OR` one element of the set B is chosen, then this is equivalent to choosing one element of the union of A and B.
```
